pipenv/environments.py

```diff
@@ -20,7 +20,7 @@
 
 # User-configuraable max-depth for Pipfile searching.
 # Note: +1 because of a temporary bug in Pipenv.
-PIPENV_MAX_DEPTH = int(os.environ.get('PIPENV_MAX_DEPTH', '3')) + 1
+PIPENV_MAX_DEPTH = int(os.environ.get('PIPENV_MAX_DEPTH', 3)) + 1
```

Comment:
@nateprewitt thank you for clarification I will put it back, I just thought that since it is intended to be a number we could have as a number for default value.